input/pagecontent/general-requirements.md

```diff
@@ -410,7 +262,96 @@ AU Core MedicationRequest|MedicationRequest.code.coding|[Australian Medication](
-- Systems **MAY** support populating and processing other terminologies but this is not a requirement of AU Core.
+- AU Core Responders **SHALL** correctly populate `Medication.code.coding` with either a code from [Australian Medication](https://healthterminologies.gov.au/fhir/ValueSet/australian-medication-1) or [PBS Item Codes](https://build.fhir.org/ig/hl7au/au-fhir-base//ValueSet-pbs-item.html), or both, if a coded value is known.
+- AU Core Requesters **SHALL** accept a Medication resource without error if `Medication.code.coding` present with either a code from [Australian Medication](https://healthterminologies.gov.au/fhir/ValueSet/australian-medication-1) or [PBS Item Codes](https://build.fhir.org/ig/hl7au/au-fhir-base//ValueSet-pbs-item.html), or both.
+
+Systems **MAY** populate and accept other code systems but this is not a requirement of AU Core.
+
+### Missing Data
+
+There are situations when information for a  data element is missing and the  system does not know reason for the absence of data. 
+
+When an element definition is *optional* (minimum cardinality = 0), including elements labelled *Must Support*, an
+- AU Core Responder **SHALL NOT** populate the element in the resource when a value is not known.  
+
+When element definition is *Mandatory* (minimum cardinality > 0), 
+- AU Core Responder **SHALL** correctly populate the element *even when* the system does not know a value or the reason a the value is absence. 
+
+The core specification provides guidance for what to do in this situation, which is summarised below.
+
+1.  For *non-coded* data elements including type [Reference](http://hl7.org/fhir/R4/references.html#Reference), 
+  - when the AU Core profile for the element does not require a sub-element
+    - use the [DataAbsentReason extension](http://hl7.org/fhir/R4/extension-data-absent-reason.html) in the data type 
+    - use the code `unknown` - The value is expected to exist but is not known.
+  
+      Example: ExplanationOfBenefit resource where the patient's insurance coverage is not available.
+      ~~~
+      {
+        "resourceType" : "ExplanationOfBenefit",
+            ...
+            "outcome" : "complete",
+              "insurance" : [
+                {
+                  "focal" : true,
+                  "coverage" : {
+                    "extension" : [
+                      {
+                        "url" : "http://hl7.org/fhir/StructureDefinition/data-absent-reason",
+                        "valueCode" : "unknown"
+                      }
+                    ]
+                  }
+                }
+              ],
+              ...
+          }
+      ~~~
+      
+  - when the AU Core profile mandates a sub-element, such as a valid Identifier or Reference data type, then the resource must contain the sub-element otherwise the resource will not be conformant,
+
+1. For *coded* data elements:
+    - *required* binding strength (CodeableConcept or code datatypes):
```

Comment:
Change to "CodeableConcept, or code datatypes" to make it consistent with other bullet point (example, preferred...bindings).

Reply:
No change. This is not desired. Could list all three code types but prefer the short hand of 'coded'.

Reply:
My comment wasn't clear enough. This was simply about adding a comma so that "(CodeableConcept or code datatypes)" changes to "(CodeableConcept, or code datatypes)". 
To make it consistent with "(CodeableConcept, or Coding datatypes)".

Reply:
removed both (CodeableConcept, or Coding datatypes) and (CodeableConcept or code datatypes) - they weren't necessary.